RealTimeFaceMaskDetector/SQLServerDLL/SQLServer.cpp

```diff
@@ -189,18 +142,6 @@ SQLServer::~SQLServer()
 	Disconnect();
 }
 
-void SQLServer::CreatePhotos(const std::vector<PhotoType>& photos)
```

Comment:
Why delete this function?

Reply:
These functions were used for the previous version of the table, when photos were writen into the table as binaries